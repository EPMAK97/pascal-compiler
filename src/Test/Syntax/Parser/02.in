var a, b : integer;
    c : array [1..10] of integer;
    d : array [1..10, 2..20] of double;
Begin
end.