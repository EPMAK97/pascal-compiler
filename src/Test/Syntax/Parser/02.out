└── Main Program
    └── var
        ├── integer
        │   ├── a
        │   └── b
        ├── array
        │   └── c
        │       └── integer
        │           ├── 1
        │           └── 10
        └── array
            └── d
                └── double
                    ├── 1
                    ├── 10
                    ├── 2
                    └── 20
